server/src/test/java/org/elasticsearch/index/query/MultiMatchQueryBuilderTests.java

```diff
@@ -476,6 +473,19 @@ public void testWithStopWords() throws Exception {
         assertEquals(expected, query);
     }
 
+    public void testDisMaxDeprecation() throws Exception {
+        String json =
+                "{\n" +
+                "  \"multi_match\" : {\n" +
+                "    \"query\" : \"foo:bar\",\n" +
+                "    \"use_dis_max\" : true\n" +
+                "  }\n" +
+                "}";
+
+        parseQuery(json);
+        assertWarnings("Deprecated field [use_dis_max] used, replaced by [use tie_breaker instead]");
```

Comment:
The syntax `[use tie_breaker instead]` is a little confusing because square brackets usually contain single names, should it just be `[tie_breaker]` or we could emit a fully custom message?

Reply:
I agree it looks slightly weird but thats what we usuall have done with ParseField deprecations in the past. "tieBreaker" is not a boolean flag so its not a direct replacements, also "tieBreaker" would otherwise act as a deprecated alias for "use_dis_max" which it isn't.
See e.g. QueryStringQueryBuilderL108 (at least on 6.x) for similar use cases where the message will contain brackets as well.

Reply:
Thanks, I now see that it's consistent with our other messages (which seems most important).